From d8da644c06e7fddf24cf16decbda0f44c4db2389 Mon Sep 17 00:00:00 2001
From: Hans-Erik Floryd <hans-erik.floryd@rt-labs.com>
Date: Wed, 23 Nov 2016 11:55:25 +0100
Subject: [PATCH] arb2: enable pcie_bus clock in EP_RC_SYS mode also

---
 drivers/pci/host/pci-imx6.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/pci/host/pci-imx6.c b/drivers/pci/host/pci-imx6.c
index 7a661dd1..3bff5b6 100644
--- a/drivers/pci/host/pci-imx6.c
+++ b/drivers/pci/host/pci-imx6.c
@@ -371,8 +371,8 @@ static int imx6_pcie_deassert_core_reset(struct pcie_port *pp)
 		goto err_pcie;
 	}
 
-	if (!IS_ENABLED(CONFIG_EP_MODE_IN_EP_RC_SYS)
-			&& !IS_ENABLED(CONFIG_RC_MODE_IN_EP_RC_SYS)) {
+	/* if (!IS_ENABLED(CONFIG_EP_MODE_IN_EP_RC_SYS) */
+	/* 		&& !IS_ENABLED(CONFIG_RC_MODE_IN_EP_RC_SYS)) { */
 		if (imx6_pcie->ext_osc) {
 			clk_set_parent(imx6_pcie->pcie_ext,
 					imx6_pcie->pcie_ext_src);
@@ -388,7 +388,7 @@ static int imx6_pcie_deassert_core_reset(struct pcie_port *pp)
 				goto err_pcie_bus;
 			}
 		}
-	}
+	/* } */
 
 	ret = clk_prepare_enable(imx6_pcie->pcie_phy);
 	if (ret) {
-- 
2.7.4

