From 8f1c5f4c074f9f3aac894a993102f218b6e09bdf Mon Sep 17 00:00:00 2001
From: Hans-Erik Floryd <hans-erik.floryd@rt-labs.com>
Date: Tue, 8 Nov 2016 10:55:43 +0100
Subject: [PATCH 2/2] arb2: do not reset board

---
 arch/arm/cpu/armv7/mx6/soc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/cpu/armv7/mx6/soc.c b/arch/arm/cpu/armv7/mx6/soc.c
index 424e869..f49476d 100644
--- a/arch/arm/cpu/armv7/mx6/soc.c
+++ b/arch/arm/cpu/armv7/mx6/soc.c
@@ -488,12 +488,14 @@ int arch_cpu_init(void)
 		 * clear the ref_ssp_en bit16 of gpr1 to workaround it.
 		 * then warm-reset imx6q/dl/solo again.
 		 */
+#if 0
 		val = readl(IOMUXC_BASE_ADDR + 0x4);
 		if (val & (0x1 << 16)) {
 			val &= ~(0x1 << 16);
 			writel(val, IOMUXC_BASE_ADDR + 0x4);
 			reset_cpu(0);
 		}
+#endif
 	}
 
 	init_aips();
-- 
2.7.4

